src/experiment_prototype/scan_classes/sequences.py

```diff
@@ -201,7 +201,7 @@ def initialize_combined_pulse_dict(pulse_timing_info):
             last_pulse_num_samps = pulse_data['total_num_samps']
 
             # If there are overlaps (two pulses within minimum separation time) then make them into one single pulse
-            min_sep = self.transmit_metadata['minimum_pulse_separation']
+            min_sep = self.transmit_metadata['min_pulse_separation']
```

Comment:
I think I preferred the longer minimum over min. Removing 'num_of' from the options attrs seems fine, but clear naming is better than shorthand. Thoughts?

Reply:
Discussed as group, decided that `min` instead of `minimum` is suitable in this instance to align with use of `max`. We should be careful to not over-simplify configuration option names in the future, however.